src/test/data/JsonAddressBookStorageTest/invalidAndValidPersonAddressBook.json

```diff
@@ -3,11 +3,19 @@
     "name": "Valid Person",
     "phone": "9482424",
     "email": "hans@example.com",
-    "address": "4th street"
+    "address": "4th street",
+    "meeting" : {
+      "start" : "",
+      "end" : ""
```

Comment:
Would it be better to include a valid meeting value here?

Reply:
I would say open another issue to add more tests for meetings, since this PR is meant to fix the current issues